packages/v4/src/content/design-guidelines/components/drawer/drawer.md

```diff
@@ -3,3 +3,13 @@ id: Drawer
 section: components
 ---
 The **Drawer** is a sliding panel that enters from the right edge of the viewport. It can be configured to either overlay content on a page or create a sidebar by pushing that content to the left.
+
+## Usage
+
+Drawers can be used to store and show more information than immediately presented on a page. A drawer with more information is usually prompted by a user click. Upon the click, the drawer comes out from the right side of the page (right to left) and shows more information, which is usually relevant to a whole page.
```

Comment:
We are currently adding a horizontal drawer in the next release so may want to update this.

Reply:
Ok, for now i've removed the "from the right side of the page (right to left)" so it's more flexible